display/ssd1306.h

```diff
@@ -732,6 +737,7 @@ class StandardDisplayController : public DisplayControllerBase<Width, col_t>, Sa
   Screen screen_ = SCREEN_STARTUP;
   Screen next_screen_ = SCREEN_UNSET;
   char message_[32];
+  bool from_sbmessage_ = false;
```

Comment:
what does this do?

Reply:
This is a prelude to having the wavlen stuff; a bypass if it's an error message. ...and now that you mention it, font.bmp syncing is prob not needed. 
Killing it.